src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -35,14 +41,19 @@ public LogicManager(Model model, Storage storage) {
         this.model = model;
         this.storage = storage;
         moduleListParser = new ModuleListParser();
+        contactListParser = new ContactListParser();
+        todoListParser = new TodoListParser();
+        parserManager = new ParserManager(moduleListParser, todoListParser, contactListParser);
     }
 
     @Override
     public CommandResult execute(String commandText) throws CommandException, ParseException {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
 
         CommandResult commandResult;
-        Command command = moduleListParser.parseCommand(commandText);
+        String commandWord = commandText.split(" ")[0];
```

Comment:
The separation of the commandWord from the respective arguments in this line could be further refined using Pattern and Matcher class.